.azurePipeline/python_init_package_install_steps.yml

```diff
@@ -14,13 +14,18 @@ steps:
       pip install -r requirements.txt
   displayName: 'Install Python requirements'
 
-- script: python setup.py sdist bdist_wheel --universal
+- script: python setup.py sdist bdist_wheel
   env:
     CC: ${{ parameters.compiler }}
-  displayName: 'Package'
+  displayName: 'Build Package'
+
+- bash: |
+      pip install dist/anonlink*.whl
+      python -c "import anonlink; print(anonlink.__version__)"
```

Comment:
Should we also run this command in the following step? (i.e. `Install anonlink for testing`)

Reply:
I don't see why, the `pytest` is much more comprehensive testing that seeing if anonlink is importable (although of course that has to be true).